localstack/services/sns/provider.py

```diff
@@ -940,6 +947,7 @@ async def message_to_subscriber(
                 traceback.format_exc(),
             )
             store_delivery_log(subscriber, False, message, message_id)
+            # check AWS format of DLQ message after application delivery failure, maybe format is wrong
```

Comment:
q: Is this a leftover from analyzing the issue?

Reply:
Sorry, yes, I didn't know how to properly test PushNotification with AWS directly, but after reading the docs it should be the same format, so it will be fixed as well.